src/main/java/edu/harvard/iq/dataverse/DatasetFieldType.java

```diff
@@ -322,10 +331,10 @@ public JsonLDTerm getJsonLDTerm() {
     public void setUri(String uri) {
     	this.uri=uri;
     }
-    
+
     /**
-     * The list of controlled vocabulary terms that may be used as values for
-     * fields of this field type.
+     * If the definition of the DatasetFieldType includes a definition for a controlled Vocabulary, i.e. a list of allowed values.
```

Comment:
This is not a complete sentence – I expect another part starting with "then", because of "If".